src/macos/mod.rs

```diff
@@ -9,10 +9,13 @@ use std::process::Command;
 use {Type, Info, Version};
 
 pub fn current_platform() -> Info {
-    let version = match retrieve() {
-        Some(v) => Version::custom(v.product_version.unwrap_or("unknown".to_string()), None),
-        None => Version::unknown(),
-    };
+    let version = retrieve().map(|x| x.product_version).map_or_else(
+        || Version::unknown(),
```

Comment:
This looks like an accidental reversion of e56d1b00891e8ac30d08b81f97b757637057f67c.

Reply:
Tank you, I'm going to fix it.